DnDTests/Services/SheetServiceTest.cs

```diff
@@ -19,35 +19,41 @@ public void OneTimeSetUp()
         {
             _attributeSettingStrategyMock = new Mock<IAttributeSettingStrategy>();
             _attributeSettingStrategyMock
-                .Setup(m => m.SetStrengthAttribute(It.IsAny<Sheet>(), It.IsAny<int>()))
+                .Setup(m => m.SetAttribute(It.IsAny<Sheet>(), It.IsAny<int>()))
                 .Returns(new Sheet());
             _service = new();
         }
 
         [Test]
-        public void SetStrengthAttribute_RollingDice_ModifyStrengthAttribute_ReturnsSheet()
+        public void SetAttribute_StrategySet_ReturnsSheet()
         {
             // Arrange
-            _service.SetStrategy(_attributeSettingStrategyMock.Object);
-            Sheet expected = new();
+            _service.SetAttributeSettingStrategy(_attributeSettingStrategyMock.Object);
 
             // Act
-            Sheet actual = _service.SetStrengthAttribute(new Sheet(), 3);
+            try
```

Comment:
take care in formatting please, use cleanup in the IDE either before pushing manually or run the code cleanup directly with a githook before pushing :)